### DISCUSSION TITLE
using the Nano IOT  and OTA update when in AP mode, any issues?

### DISCUSSION THREAD

**Discussion by sdetweil** [JAndrassy/ArduinoOTA#98] [General]
all of the examples show the arduino connecting to some other network.  in our situation there is no guaranty of wifi nearby, so need to be the AP..  and connect from the central device to the AP. 

there shouldn't be any difference once the wifi connection is completed, right? 
start a server, web maybe, send a form to request the 'file', have user navigate to the file on their  connected device, and hit upload (form button)

update from file and then reboot and go back to  non-AP mode, waiting for next central connect.

**Comment by JAndrassy** [comment]
this library has 3 functions. the main function is to store and apply the updated binary and it doesn't matter how is the binary transported. 

Next function of the library is a TCP server to receive upload of the binary from the Arduino's ardinoOTA upload tool used by the IDE. 

And finally the third supporting function is the propagation of the 'network port' for Arduino IDE using MDNS.

The library doesn't have upload  over a web form.

**Reply by sdetweil** [reply]
understand..  working on the form approach, to get the file transferred.. then use the apply function
